cpp/ql/lib/semmle/code/cpp/dataflow/internal/SubBasicBlocks.qll

```diff
@@ -7,6 +7,8 @@
 // non-monotonic recursion errors in queries that use both the data flow
 // library and the `SubBasicBlocks` library.
 /**
+ * DEPRECATED: Use `semmle.code.cpp.dataflow.new.DataFlow` instead.
```

Comment:
As in, via the identical-files check?  If that's the case perhaps I should exclude it from the check, rather than make it look like it isn't deprecated???

Reply:
> As in, via the identical-files check?

Yes.

> If that's the case perhaps I should exclude it from the check, rather than make it look like it isn't deprecated???

It's an internal file, so it not being marked as deprecated shouldn't matter. Also as long as they're in identical-files, any fix we'll do on one side will propagate to the other side. That's probably a property we want to keep.

Reply:
Well, the library is deprecated, but this file certainly isn't. It just occurs in a deprecated context. So anything you would fix here - accidental or not - will propagate to the version of the file we're definitely keeping.